layers/cmd/ingest_json.go

```diff
@@ -26,6 +27,67 @@ import (
 	"github.com/cloudprivacylabs/opencypher/graph"
 )
 
+type JSONIngester struct {
+	BaseIngestParams
+	ID           string
+	InitialGraph string
+}
+
+func (ji JSONIngester) Run(pipeline *PipelineContext) error {
+	initialGraph := ji.InitialGraph
```

Comment:
Leave initial graph loading to pipeline initialization. Look at the pipeline comments.